tests/Oryx.Integration.Tests/Node/NodeSqlServerIntegrationTests.cs

```diff
@@ -80,5 +83,30 @@ private async Task Run_NodeApp_MicrosoftSqlServerDBAsync(string imageTag)
                 });
         }
 
+        protected string GetKeyvaultSecretValue(string keyvaultUri, string secretName)
+        {
+            var client = new SecretClient(new Uri(keyvaultUri), new DefaultAzureCredential());
+            var sasToken = client.GetSecret(secretName).Value.Value;
+            return sasToken;
+        }
```

Comment:
nit: let's separate the end of one method and the start of the next with a new line